.github/workflows/ocwm-issue-collector.yml

```diff
@@ -0,0 +1,86 @@
+name: Search for issues and PR labeled 'agenda' and add them to the agenda
+
+on:
+  schedule:
+    - cron: '0 0 * * 0'  # Runs every Sunday at midnight
+
+  repository_dispatch:
+    types: add-issues-ocwm
+
+jobs:
+  search_and_add:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install dependencies
+      run: npm install @octokit/core
+    - name: Adding Issues
+      uses: actions/github-script@v6
+      env:
+        PLACEHOLDER: '<!-- | [TOPIC] [IssuePRDiscussion] | [owner] | -->'
+        OWNER: ${{ vars.ORGANISATION }}
+        REPO: 'community'
+        REPO_NAMES: ${{ vars.REPOSITORIES }}
+        LABELS: ${{ vars.LABELS }}
+        MY_TOKEN: ${{ secrets.AUTH_TOKEN }}
+      with:
+        script: |
+          try {         
+            const repoNames = process.env.REPO_NAMES;
+            const labels = process.env.LABELS;
+            const placeholder = process.env.PLACEHOLDER;
+            console.log("Placeholder:" + placeholder);
+
+            const octokit = require('@octokit/core').Octokit  
+            const mygithub = new octokit({ auth: process.env.MY_TOKEN })
+
+            let repositories = repoNames.split(",")
+            let targetLabel = encodeURIComponent(labels.split(',')[0])
+            let appendLabel = encodeURIComponent(labels.split(',')[1])
+            console.log("Repositories:" + repositories);
+            for (let r = 0; r < repositories.length; r++) {
+              console.log(`GET /repos/${process.env.OWNER}/${repositories[r]}/issues?labels=${targetLabel}`);
+              console.log(`GET /repos/${process.env.OWNER}/${repositories[r]}/issues?labels=${appendLabel}&per_page=1`);
```

Comment:
Is `${process.env.OWNER}/${repositories[r]}` right?  In the actual call (line 50), you have `${{env.OWNER}}/${{env.REPO}}`.